data_set/taskset/sanity_check.md

```diff
@@ -0,0 +1,276 @@
+- `find . -type f -exec cat {} \; -exec cat {} \; -exec cat {} \; -exec cat {} \; -exec cat {} \;`
+
+Helpful
+
+## 3. https://www.commandlinefu.com/commands/view/1295/quickly-rename-a-file (Quickly rename a file)
+One-liner: `mv filename.{old,new}`
+
+### Prompt: Quickly rename a file
+- `find . -type f -exec ls -s {} \; | xargs -0 -I {} rm {}`
+- `find . -type f -exec ls -s {} \; | xargs -0 -I {} rename 's/ /_/g' {}`
+- `find . -type f -exec ls -s {} \; | sort -n | head -n quickly`
+
+Unhelpful
+
+### Adapted prompt: Rename index.txt to index.csv
+- `mv index.txt index.csv`
+- `rename 's/\.txt/index.txt' index.csv`
+- `rename 's/\.txt/.txt/' index.csv`
+
+Helpful
+
+## 4. https://stackoverflow.com/questions/1529946/ (Linux: copy and create destination dir if it does not exist)
+One-liner: `mkdir -p "$d" && cp file "$d"`
+
+### Prompt: Copy and create destination dir if it does not exist
+- `cp -n`
+- `cp -r dir_dir dir1`
+- `cp -R dir /path/to/dir`
+
+Helpful (Copy directory)
+
+### Adapted prompt: Copy folder src in new folder a/b
+- `find a/b -type d -exec mkdir -p {} \;`
+- `find a/b -type d -print0 | xargs -0 -I {} cp -t a/b {}`
+- `find a/b -type d -print0 | xargs -0 -I {} cp a/b {}`
+
+Helpful (`mkdir -p`)
+
+## 5. https://stackoverflow.com/questions/3137094/
+One-liner: `wc -l <filename>`
+How to count lines in a document?
+
+### Prompt: How to count lines in a document?
+- `cat /dev/urandom | tr -d -c '[0-9]+' | wc -c`
+- `cat /dev/urandom | tr -d -c '[0-9] ' | wc -c`
+- `cat /dev/urandom | tr -d -c '[a-z]' | wc -c`
+
+Unhelpful
+
+### Adapted prompt: Count lines in index.html
+- `wc -l index.html`
+- `cat index.html | wc -l`
+- `wc index.html`
+
+Helpful
+
+## 6. https://superuser.com/questions/482435/
+One-liner: `find  . -name 'name*' -exec rm {} \;`
+
+### Prompt: How to remove all files starting with a certain string in Linux
+- `find . -type f -exec grep "linux" {} \; -exec rm {} \;`
+- `find /usr/share/linux -type f -exec grep "linux" {} \; -exec rm {} \;`
+- `find /tmp -type f -exec grep "linux" {} \; -exec rm {} \;`
+
+Helpful
+
+### Adapted prompt: Remove all files starting with "prefix"
+- `find . -name 'prefix*' -exec rm {} \;`
+- `find . -type f -name 'prefix*' -exec rm {} \;`
+- `find . -name "prefix*" -exec rm {} \;`
+
+Helpful
+
+## 7. https://stackoverflow.com/questions/4997693/
+One-liner: `diff --brief --recursive dir1/ dir2/`
+
+### Prompt: Given two directory trees, how can I find out which files differ by content?
+- `find out -type f -exec file {} \;`
+- `find out -type f -user ? -exec ls -l {} \;`
+- `find out -type f -exec md5 {} \;`
+
+Not helpful
+
+### Adapted prompt: Compare different files in directory foo and bar
+- `diff foo bar`
+- `find foo bar -type f -exec diff {} {} \; -exec diff {} {} \; -exec diff {} {} \;`
+- `find foo bar -type f -exec diff {} {} \; -exec diff {} {} \; -exec diff {} {} \; -exec diff {} {} \;`
+
+Helpful
+
+## 8. https://unix.stackexchange.com/questions/144208/
```

Comment:
Uhhh. I guess just try to stay consistent either way. Since the rest of them don't seem to have the prompt in the section header either.

Reply:
I fixed all the inconsistencies. I thought I did but apparently not. Thank you for calling it out!